### DISCUSSION TITLE
Migrating to text2confl

### DISCUSSION THREAD

**Discussion by dgautier** [md2conf/md2conf#202] [General]
Hi @qwazer ,

What do you think about working together in the [text2confl ](https://github.com/text2confl/text2confl) repo ? Those tools are quite the same and it seems to me working together could be more efficient than maintaining multiple tools. Also i'm not the author of this and trying to get @zeldigas to join. 

Cheers,

**Comment by qwazer** [comment]
Hi @dgautier 

Thanks for your question and interest to the md2conf project. Frankly speaking, I'm not interested in such union of efforts at this moment.  Reasons:
1) I'm too busy with other projects.
2) It's my single man hobby project without any commitment to the community. 

My roadmap is: 1) polish and finish `dump` functionality (I don't know in this year or not) 2) Try to pass the project to the community. I made the same with my another project https://github.com/qwazer/markdown-confluence-gradle-plugin. Now it supported by community members, not by me.

BTW, my respect to @zeldigas. [text2confl](https://github.com/zeldigas/text2confl) is a really nice tool. We discussed our visions here https://github.com/md2conf/md2conf/issues/126.

**Comment by dgautier** [comment]
Hi,  

It's a bit sad that you can't join cause i really think  those tools have the same goal just some different approaches and joining efforts would have been really nice.

I'd like to merge both your md2conf and text2confl in a single tool. I still hope @zeldigas will join. We'll see if it makes any progress in the future, 

Thanks for the work you've already done.
